e2e/test/prerequisites/E2ETestsSetup/e2eTestsArmTemplate.json

```diff
@@ -51,6 +51,22 @@
             "metadata": {
                 "description": "The name of the DPS used by tests."
             }
+        },
+        "TestGetFunctionName": {
```

Comment:
remove this, don't need a TestGet function for our tests

Reply:
I did not know that :) Will remove.